rov/index.html

```diff
@@ -70,14 +70,14 @@ <h2 class="fs-1">Creating robots for the most extreme environments.</h2>
                             </p>
                         </div>
                         <div class="col-md-5">
-                            <img class="rounded img-fluid" src="./images/2021_Team_Competition.jpg" alt="2021 Competition Team">
```

Comment:
You may want to purge old photos to avoid having a bunch in the git repo

Reply:
problem for later